src/components/MakeWidget/subcomponents/RateField/helpers/customStringToSignificantDecimals.ts

```diff
@@ -0,0 +1,48 @@
+export const customStringToSignificantDecimals: (
```

Comment:
This works: 
```
it("Should only ignore decimal leading zeroes if number is less than 0", () => {
    expect(customStringToSignificantDecimals("1234.000056789", 5)).toBe("1234.00005");
    expect(customStringToSignificantDecimals("1234.000056789", 4)).toBe("1234.0000");
  });
```

It will always trim to sigDecimals input when > 1. The change I made was previously, if it was > 1 it would automatically trim to 4 decimal places:

````trimmedDecimals = afterDecimalPoint.slice(0, 4);````

Reply:
Yeah add a condition to trim out unnecessary zeroes.